app/theme/client/imports/components/sidebar/toolbar.css

```diff
@@ -110,14 +110,39 @@
 	}
 }
 
-@media (width <= 400px) {
-	.toolbar {
-		padding: 0 var(--sidebar-extra-small-default-padding) var(--sidebar-small-default-padding);
-	}
-}
-
 .rtl .toolbar {
 	& .rc-input__icon + .rc-input__element {
 		padding: 0.5rem 2.25rem 0.5rem 1rem;
+		line-height: 4.5em !important;
 	}
 }
+
+@media (width <= 500px) {
+	.toolbar {
+		height: 36px !important;
+		margin: -3px -10px;
+		padding: 0 var(--sidebar-extra-small-default-padding) var(--sidebar-small-default-padding);
+
+		& .rc-input {
+			&__wrapper {
+				padding: 0;
+				color: var(--rc-color-primary-light);
+				text-align: center;
+			}
+			&__icon {
+				left: 2rem;
+				&--right {
+					display: none !important;
+				}
+			}
+
+		}
+		& .rc-input__icon + .rc-input__element {
+			padding: 0.5rem 2.25rem 0.5rem 3rem;
+			height: 40px !important;
+			border-color: #81868d;
+			border-radius: 15px;
+			width: 90%;
+		}
+	}
+}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/manual/RocketChat/Rocket-Chat/pullRequest?prid=4892745)